pkg/baseline/packageIndex.json

```diff
@@ -9,8 +9,9 @@
       "BaselineVersion": "4.4.0",
       "InboxOn": {},
       "AssemblyVersionInPackageVersion": {
-        "4.1.1.0": "4.4.0",
-        "4.1.2.0": "4.4.0"
+        "4.1.1.0": "4.5.0",
```

Comment:
Thanks @ericstj. Based on this 4.1.1.0 should be mapped to 4.3.0 package version as 4.1.1.0 was first released in 4.3.0. @StephenBonikowsky can you try to see if this solves the validation error?

> You shouldn't be changing a mapping unless you decided not to ship that assembly in the previous release.

@ericstj, just to understand, if an assembly is already released, how can I decide not to ship it in a *previous* release?